spec/spec_helper_acceptance.rb

```diff
@@ -1,35 +1,50 @@
-require 'beaker-rspec'
-require 'beaker-puppet'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
+require 'serverspec'
+if ! Gem.win_platform?
+  require 'beaker-rspec'
+  require 'beaker-puppet'
+  require 'beaker/puppet_install_helper'
+  require 'beaker/module_install_helper'
 
-run_puppet_install_helper
-install_module_dependencies
-install_module
-collection = ENV['BEAKER_PUPPET_COLLECTION'] || 'puppet5'
+  run_puppet_install_helper
+  install_module_dependencies
+  install_module
+  collection = ENV['BEAKER_PUPPET_COLLECTION'] || 'puppet5'
+end
 
-UNSUPPORTED_PLATFORMS = ['Suse','AIX','Solaris']
+if Gem.win_platform?
+  set :backend, :cmd
+end
+
+if Gem.win_platform?
+  def fact(v)
```

Comment:
Normally beaker provides the `fact` function but we can't load beaker for tests run on Windows in appveyor.  We use `fact('osfamily')` in some places and to avoid undefined method errors I create the function just when running on Windows (appveyor).

Reply:
could you document that so it's clear why we have the function

Reply:
Yep, added a comment.